src/hyperloglog.c

```diff
@@ -36,6 +36,9 @@
 #include <math.h>
 
 #ifdef HAVE_AVX2
+/* Define __MM_MALLOC_H to prevent importing the memory aligned
+ * allocation functions, which we don't use. */
+#define __MM_MALLOC_H
```

Comment:
Do all compilers use the same name for this macro or is it just clang?

Reply:
If it's only for clang, then it feels like a workaround just for that, but let's merge this for now to silence the CI.